securedrop/source_templates/base.html

```diff
@@ -26,6 +26,9 @@
       {% endblock %}
 
       <div class="panel selected">
+	{% if 'logged_in' in session %}
+	<a href="{{ url_for('logout') }}" class="btn pull-right">Log Out</a>
+	{% endif %}
```

Comment:
I originally had it outside the .panel. It shifted the entire page down a button size; I felt that it looked very ugly.